### PR TITLE
Adds the 2024 American bitcoin survey to the research page

### PR DESCRIPTION
**Pull request by GBKS**
Small addition to the research page.

🤜[Check the preview](https://deploy-preview-1105--bitcoin-design-site.netlify.app/guide/resources/design-research/#independent-research)🤛

### CODE REVIEW

File: guide/resources/design-research.md [hunk 1]
```diff
@@ -97,6 +97,12 @@ A survey and follow-up interview of 990 bitcoin users to determine bitcoin manag
 
 ## Independent research
 
+#### [2024 American Bitcoin Survey report](https://www.thenakamotoproject.org/report)
+
+_2024 by The Nakamoto Project_
```
Comment (yashrajd) [inline guide/resources/design-research.md:102]:
```suggestion
 Understanding Bitcoin Adoption in the United States: Politics, Demographics, & Sentiment
```

**Review by yashrajd — CHANGES_REQUESTED** [review]
The title of the report seems to be as below. I wasn't able to find the title "2024 American Bitcoin Survey report" or something similar to it in the document. Please reconsider.

File: guide/resources/design-research.md [hunk 2]
```diff
@@ -97,6 +97,12 @@ A survey and follow-up interview of 990 bitcoin users to determine bitcoin manag
 
 ## Independent research
 
+#### [2024 American Bitcoin Survey report](https://www.thenakamotoproject.org/report)
```
Comment (yashrajd) [inline guide/resources/design-research.md:100]: Should we link to the actual report link itself? Not sure if this is correct though...
```suggestion
#### [2024 American Bitcoin Survey report](https://cdn.prod.website-files.com/6644db90e58a1fbeb1c70ddc/669f69702fa1b4b3bf09dbac_TNP%20Q3%202024%20Report%20(Digital).pdf)
```

Comment (GBKS) [reply guide/resources/design-research.md:100]: I understand it's nice to go around their email form, but I would honor how their decisions and keep the link to the report landing page.

File: guide/resources/design-research.md (hunk 1 above)
Comment (GBKS) [reply guide/resources/design-research.md:102]: It's from the report landing page.

image

Comment (yashrajd) [reply guide/resources/design-research.md:102]: Yes I see that, it seems to be an example of unintentional formatting and/or poor writing. I would still encourage using the suggested title "Understanding Bitcoin Adoption in the United States: Politics, Demographics, & Sentiment" for following reasons:
- This title appears on the first page of the report (see image below)
- It is more descriptive, precise and accurate characterisation of the research
- It mentions that this was not a general bitcoin survey but centred around adoption
- NP do not seem to have performed this research in 2023 or earlier

Screenshot 2024-08-05 at 11 38 24

**Review by mouxdesign — APPROVED** [review]
Nice addition, LGTG.

**Pull request merged**